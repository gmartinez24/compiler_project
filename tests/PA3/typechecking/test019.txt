// Alex Garza
// Testing function call with missing parameters
main

int a, b;

{
    a = 1;
    b = 2;
    call printInt();

    call println(a);

    call printBool();
}.