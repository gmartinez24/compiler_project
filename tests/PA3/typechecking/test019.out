Error type-checking file.
TypeError(10,17)[ErrorType(Call with args TypeList() matches no function signature.)]
TypeError(12,16)[ErrorType(Call with args TypeList(int) matches no function signature.)]
TypeError(14,18)[ErrorType(Call with args TypeList() matches no function signature.)]